tests/test_detectAndMeasure.py

```diff
@@ -705,6 +705,16 @@ def test_mask_streaks(self):
         # Check that the entire image was not masked STREAK
         self.assertFalse(np.all(streakMaskSet))
 
+    def test_trailed_glints(self):
```

Comment:
It would be good to test that `writeGlintInfo` provides the expected outputs as well.